Makefile

```diff
@@ -94,7 +94,7 @@ k8s-test:
 	#make ssh-build 
 	cd vagrant/k8s/ && CONTIV_K8=1 vagrant ssh k8master -c 'sudo -i bash -lc "cd /opt/gopath/src/github.com/contiv/netplugin && make run-build"'
 	CONTIV_K8=1 cd vagrant/k8s/ && ./start_sanity_service.sh
-	CONTIV_K8=1 CONTIV_NODES=3 godep go test -v -timeout 540m ./systemtests -check.v -check.f "00SSH|Basic|Network|Policy|TestTrigger|ACIM|HostBridge" 
+	CONTIV_K8=1 CONTIV_NODES=3 go test -v -timeout 540m ./test/systemtests -check.v -check.f "00SSH|Basic|Network|Policy|TestTrigger|ACIM|HostBridge" 
```

Comment:
sure i will need to modify the systemtests for that to derive node number. I can do it in the next pr. btw the vagrantfile used here is k8s.